packages/docs/concepts/lowdefy-schema.yaml

```diff
@@ -24,204 +24,262 @@ _ref:
-                            pageId: page3
-                            properties:
-                              title: Page 3
-              - id: page-1-and-3
-                links:
-                  - id: page1
-                    type: MenuLink
-                    pageId: page1
-                    properties:
-                      title: Page 1
-                      icon: AiOutlineFile
-                  - id: page3
-                    type: MenuLink
-                    pageId: page3
-                    properties:
-                      title: Page 3
-                      icon: AiOutlineControl
-            ```
+            [TODO]: <> (# TODO: Add plugins things here)
+
+            ## Menus
+
+            Menu objects are objects with links to pages. They are filtered to only show pages that the user is authorized to see and are used by blocks such as `PageSiderMenu` to render the links in the menu. If no menus are provided, a default menu is created, with links to all of the defined pages, and with their pageIds as menu item titles.
+
+            See more about how menu objects are used and defined [here](/menus).
 
-            # Pages
+            ## Pages
 
             Pages in a Lowdefy app are actually just blocks, the building blocks of a Lowdefy app, with a few extra restrictions and features.
 
             Each page should have an `id` that is unique among all pages in the app. Each page is served with the `pageId` as the url route. That is, if you create a page with id `page1`, it will be served at `domain.com/page1`.
 
-            Page blocks should be of category `context`, in order to create a context for the page. This means they always have the context initialization events like `onInit` to initialize the page.
+            [TODO]: <> (# TODO: Below will change once we add the head configuration option on pages. Discuss that here.)
 
             If `properties.title` is set on a page block, the title will be set as the page title (This is the title displayed on the tabs in your browser).
 
-            # References and templates
-
-            References are used to split the configuration of an app into logically distinct files, and to reuse configuration in the app. References can be used almost anywhere in the configuration, as long as the configuration remains valid YAML.
-
-            References are made using the [`_ref`](/_ref) operator. If the referenced file has a `.yaml` or `.json` extension, the contents of the file will be parsed, else the file content is included as a string (this is useful for `.md` or `.html` files). As an example of splitting an app into logically distinct files, references can be used to write each page as a separate file:
+            Let's have a look at how to define a page and it's blocks. We can start with a simple page a card block on it. We can add a title block and a paragraph block to our card so that it won't be so empty.
```

Comment:
We try not to avoid using first person or personal pronouns in the docs. Although tricky to keep it consistent. This could read:

To define a page and it's blocks. Start with a simple page with a card block on it. Then a title block and a paragraph block can be added to the card so that it won't be so empty.

Reply:
Although, maybe not always consistent on this. Like in the tutorial and how-tos, it feels more natural to use personal pronouns.. What do you think?

Reply:
I actually like how this reads. It feels less stiff and overly formal

Reply:
@JohannMoller Yeah. I also like that.. Think we should just decide on a standard and then try to consistently implement. Maybe the more formal descriptions reference with informal examples and how-tos is a nice balance?